tango/sweeps.py

```diff
@@ -0,0 +1,68 @@
+import itertools
+import json
+import subprocess
+from collections import OrderedDict
+from dataclasses import dataclass
+
+from tango.common import Params, Registrable
+
+
+class Sweeper(Registrable):
+    def __init__(
+        self,
+        main_config_path: str,
+        sweeps_config_path: str,
+        components: str,
```

Comment:
The `Sweeper` class needs to be aware of the `Workspace` being used in order to get the results of each target step, and to pass the workspace URL when it runs each experiment (with the `-w` option).